app/src/main/java/com/example/woloxapp/MainActivity.kt

```diff
@@ -1,11 +1,23 @@
 package com.example.woloxapp
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import com.example.woloxapp.ui.login.LoginFragment
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        supportActionBar?.hide()
+        getWindow().setFlags(
+            WindowManager.LayoutParams.FLAG_TRANSLUCENT_NAVIGATION,
+            WindowManager.LayoutParams.FLAG_TRANSLUCENT_NAVIGATION
```

Comment:
We use the Kotlin scope function "with" for resume code using the context of this function.
This code should be:
`with(WindowManager.LayoutParams) {
  getWindow().setFlags(
               FLAG_TRANSLUCENT_NAVIGATION,
               FLAG_TRANSLUCENT_NAVIGATION
  )
}`

Read about Kotlin scope functions, are very interesting.